File: library/python/auth_client_parser/examples/token.py
import auth_client_parser


def get_uid_from_token():
    token = "AQAAAADue-GoAAAI2wSUpddcm0BIk-r70vL1O_A"

    oauth_token = auth_client_parser.OAuthToken(token)
    assert oauth_token.ok
    return oauth_token.uid

File: library/cpp/auth_client_parser/examples/token.cpp
#include <library/cpp/auth_client_parser/oauth_token.h>

#include <util/generic/strbuf.h>

namespace NExample {
    ui64 GetUidFromToken() {
        TString token = "AQAAAADue-GoAAAI2wSUpddcm0BIk-r70vL1O_A";
        NAuthClientParser::TOAuthToken oAuthToken;
        Y_ENSURE(oAuthToken.Parse(token));
        return oAuthToken.Uid();
    }
}
